frontend/src/scenes/annotations/Annotations.tsx

```diff
@@ -83,7 +84,27 @@ export function Annotations(): JSX.Element {
             },
             sorter: (a, b) => annotationScopeToLevel[a.scope] - annotationScopeToLevel[b.scope],
         },
-        createdByColumn() as LemonTableColumn<AnnotationType, keyof AnnotationType | undefined>,
+        {
+            title: 'Created by',
+            dataIndex: 'created_by',
+            render: function Render(_: any, item) {
+                const { created_by, creation_type } = item
+                return (
+                    <div className="flex flex-row items-center">
+                        <ProfilePicture
+                            name={creation_type === 'GIT' ? 'GitHub automation' : created_by?.first_name}
+                            email={creation_type === 'GIT' ? 'maxbot@posthog.com' : created_by?.email}
```

Comment:
`maxbot@posthog.com`, I think it might be better to just omit the email in this case

Reply:
<img width="697" alt="Screenshot 2023-02-09 at 16 18 06" src="https://user-images.githubusercontent.com/984817/217872722-9ba6ab32-a676-43cf-9ca8-b3f443374447.png">

Was wondering about setting up gravatar to have a nice profile image but there's an easier way